website/docs/r/cluster.html.markdown

```diff
@@ -10,18 +10,26 @@ description: |-
 
 Provides RKE cluster resource. This can be used to create RKE clusters and retrieve their information.
 
+RKE clusters can be configured in distinct ways:
```

Comment:
RKE clusters can be defined in the provider:

Reply:
Updated